docs/tutorials/init_cfg.md

```diff
@@ -0,0 +1,159 @@
+
+   NOTE: Value of `layer` key is the class name with attributes weights and bias of Pytorch, (so such as  `MultiheadAttention layer` is not supported).
+
+- Define `layer` key for initializing module with same configuration.
+
+  ```python
+  init_cfg = dict(type='Constant', layer=['Conv1d', 'Conv2d', 'Linear'], val=1)
+  # initialize whole module with same configuration
+  ```
+
+-  Define `layer` key for initializing layer with different configurations.
+
+  ```python
+  init_cfg = [dict(type='Constant', layer='Conv1d', val=1),
+              dict(type='Constant', layer='Conv2d', val=2),
+              dict(type='Constant', layer='Linear', val=3)]
+  # nn.Conv1d will be initialized with dict(type='Constant', val=1)
+  # nn.Conv2d will be initialized with dict(type='Constant', val=2)
+  # nn.Linear will be initialized with dict(type='Constant', val=3)
+  ```
+
+2. Initialize model by `override` key
+
+- When initializing some specific part with its attribute name, we can use `override` key, and the value in `override` will ignore the value in init_cfg.
+
+  ```python
+  # layers：
+  # self.feat = nn.Conv1d(3, 1, 3)
+  # self.reg = nn.Conv2d(3, 3, 3)
+  # self.cls = nn.Linear(1,2)
+
+  init_cfg = dict(type='Constant',
+                  layer=['Conv1d','Conv2d'], val=1, bias=2,
+                  override=dict(type='Constant', name='reg', val=3, bias=4))
+  # self.feat and self.cls will be initialized with 	dict(type='Constant', val=1, bias=2)
+  # The module called 'reg' will be initialized with dict(type='Constant', val=3, bias=4)
+  ```
+
+- If `layer` is None in init_cfg, only sub-module with the name in override will be initialized, and type and other args in override can be omitted.
+
+	```python
+	# layers：
+	# self.feat = nn.Conv1d(3, 1, 3)
+	# self.reg = nn.Conv2d(3, 3, 3)
+	# self.cls = nn.Linear(1,2)
+
+	init_cfg = dict(type='Constant', val=1, bias=2, 	override=dict(name='reg'))
+
+	# self.feat and self.cls will be initialized by Pytorch
+	# The module called 'reg' will be initialized with dict(type='Constant', val=1, bias=2)
+	```
+
+- If we don't define `layer` key or `override` key, it will not initialize anything.
+- Invalid usage
+
+	```python
+	# It is invalid that override don't have name key
+	init_cfg = dict(type='Constant', layer ['Conv1d','Conv2d'], val=1, bias=2,
+	            	override=dict(type='Constant', val=3, bias=4))
+
+	# It is also invalid that override has name and other args except type
+	init_cfg = dict(type='Constant', layer ['Conv1d','Conv2d'], val=1, bias=2,
+	                override=dict(name='reg', val=3, bias=4))
+	```
+
+3. Initialize model with the pretrained model
+
+	```python
+	init_cfg = dict(type='Pretrained',
+                checkpoint='torchvision://resnet50')
+	```
+
+More details can refer to [cnn.md](https://github.com/open-mmlab/mmcv/blob/master/docs/cnn.md) and [Pull-Request](https://github.com/open-mmlab/mmcv/pull/780)
```

Comment:
https://mmcv.readthedocs.io/en/latest/cnn.html#weight-initialization would be better